arch/arm/boot/dts/bcm283x.dtsi

```diff
@@ -565,6 +565,34 @@
 			status = "disabled";
 		};
 
+		csi0: csi0@7e800000 {
+			compatible = "brcm,bcm2835-unicam";
+			reg = <0x7e800000 0x800>,
+			      <0x7e802000 0x4>;
+			interrupts = <2 6>;
+			clocks = <&clocks BCM2835_CLOCK_CAM0>;
+			clock-names = "lp";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			#clock-cells = <1>;
+
```

Comment:
For upstreaming these empty lines should be avoided

Reply:
Removed.

Reply:
Maybe there was a misunderstanding. I wasn't talking about the cells properties. I meant really the empty lines in the csi0 and csi1 node.

Reply:
OK. Although I see a fair number of blank lines in gpio, i2s, dsi0, dsi1.
Another update incoming.